README.md

```diff
@@ -38,9 +38,9 @@ The following properties can be configured:
 - `SCENARIO_DIRECTORY`
     - default is `swagger_server/itm/data/%(EVALUATION_TYPE)s/scenarios/`
 - `SOARTECH_URL`
-    - default is `http://10.216.38.125:8084`
+    - default is `http://10.216.38.25:8084`
```

Comment:
It looks like this needs to change from 125 to 25 in the config template.

Reply:
ahh good catch, done...

Reply:
My search and replace skills are crap today... done!